### ISSUE TITLE
Unable to commit only submodule updates

### ISSUE THREAD

**Issue by lencioni** [causes/overcommit#148]
In my dotfiles repo, I am trying to update some git submodules, but when I do it, overcommit fails out and I am unable to commit.

```sh
~/dotfiles git:master ❯❯❯ git commit
Running pre-commit hooks

✓ All pre-commit hooks passed

Unable to restore working tree after pre-commit hooks run:
STDOUT:
STDERR:No stash found.
```

At the end of this, I am left with my updated submodules uncommitted and unstaged. This might be happening because overcommit is creating a stash with nothing in it (because submodules are weird), which results in no stash getting created.

I believe I used to be able to do this, so I think this might be a problem with the latest release. overcommit 0.23.0

**Comment by lencioni** [comment]
As a temporary workaround, I just did this:

```sh
overcommit --uninstall; git commit; overcommit --install
```

**Comment by jawshooah** [comment]
Worse still, it ignores the fact that no stash was created and applies the most recent stash anyway!

**Comment by sds** [comment]
Fixed by #149. Thanks!
